src/com/senlainc/warsaw/tyurin/Test.java

```diff
@@ -0,0 +1,47 @@
+package com.senlainc.warsaw.tyurin;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class Test {
+
+    public static void main(String[] args) {
+
+        Garage garage = Garage.getInstance();
+
+        System.out.println("Create new order");
+        Order order = new Order();
+        garage.addOrder(order);
+
+        System.out.println("Close order");
```

Comment:
Changing local order (not from garage) will work here because of the same link to the object, but in other cases you should always try to receive object to change from it holder.
Read more about java approach to pass variables.